bindings/Python/py11glue.cpp

```diff
@@ -51,10 +51,18 @@ struct type_caster<adios2::py11::MPI4PY_Comm>
      */
     bool load(handle src, bool)
     {
+        // Import mpi4py if it does not exist.
+        if (!PyMPIComm_Get)
+        {
+            if (import_mpi4py() < 0)
+            {
+                throw std::runtime_error(
+                        "ERROR: mpi4py not loaded correctly\n"); /* Python 2.X */
+            }
+        }
```

Comment:
I think I was involved in writing that caster, but I never fully understood the `import_mpi4py` thing.
Why is importing mpi4py still necessary at that point? I would think if the user is passing a communicator from python code, they must already have imported mpi4py themselves, or does that not propagate through into the wrapping layer?